test/itkBSplineScatteredDataPointSetToGradientImageFilterTest.cxx

```diff
@@ -58,16 +58,15 @@ int itkBSplineScatteredDataPointSetToGradientImageFilterTest( int argc, char *ar
     }
 
 
-  typedef itk::Mesh< VectorImageType::PixelType, Dimension > MeshType;
-  typedef InputImageType                                     OutputImageType;
+  using MeshType = itk::Mesh< VectorImageType::PixelType, Dimension >;
+  using OutputImageType = InputImageType;
 
-  typedef itk::ImageToPointSetFilter< VectorImageType, MeshType >
-    ImageToPointSetType;
+  using ImageToPointSetType = itk::ImageToPointSetFilter< VectorImageType, MeshType >;
   ImageToPointSetType::Pointer imageToPointSet = ImageToPointSetType::New();
   imageToPointSet->SetInput( 0, imageToVI->GetOutput() );
 
-  typedef itk::BSplineScatteredDataPointSetToGradientImageFilter< MeshType,
-                                                                  PixelType >  PointSetToGradientFilterType;
+  using PointSetToGradientFilterType = itk::BSplineScatteredDataPointSetToGradientImageFilter< MeshType,
```

Comment:
Break line at `<`, `=` or have no line break?

Reply:
Done in c577d13.